tests/fixtures/InitCommandTest/default_readme.md

```diff
@@ -0,0 +1,52 @@
+# My App
+
+Project description will be here
+
+## Project Resources & Contacts
```

Comment:
let's add blank lines before and after headers

Reply:
That's test results